libs/blocks/global-navigation/global-navigation.js

```diff
@@ -621,6 +621,8 @@ class Gnav {
       children: getChildren(),
     });
 
+    // Exposing UNAV config for consumers to modify certain values/functions and reload UNAV if needed
+    CONFIG.universalNav.universalNavConfig = getConfiguration();
     window.UniversalNav(getConfiguration());
 
     isDesktop.addEventListener('change', () => {
```

Comment:
Would you want to change the `reload` inside this event listener callback to use `CONFIG.universalNav.universalNavConfig`? The user changing the viewport might overwrite your config during the unav reload.

Reply:
that would still call getConfiguration() as it does today (same as window.UniversalNav(getConfiguration()). the only point of this is to add it to the CONFIG object so that it can be exported and consumed by others as needed.

Reply:
My understanding is that you'll be making changes to the object referenced by `CONFIG.universalNav.universalNavConfig` but `getConfiguration` will generate a new object with it's own reference that won't contain any changes you made to `CONFIG.universalNav.universalNavConfig`, so this new object could "overwrite" the previous config you reloaded unav with.